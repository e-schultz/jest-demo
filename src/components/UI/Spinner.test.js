import { render } from '@testing-library/react'
import Spinner from './Spinner'

describe('Spinner should render correctly', () => {
    it('should match snapshot', () => {
        const { container } = render(<Spinner />)
        expect(container.firstChild).toMatchSnapshot()
    })
})
