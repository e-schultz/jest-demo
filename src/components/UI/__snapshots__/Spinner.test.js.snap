// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should render correctly should match snapshot 1`] = `
<div
  class="css-myp9de"
>
  Loading...
</div>
`;
